mouse-look.js

```diff
@@ -43,6 +45,14 @@ WL.registerComponent('mouse-look', {
             }
         }.bind(this));
 
+        if(this.pointerLockOnClick){
+            WL.canvas.addEventListener('mousedown', function() {
```

Comment:
`() => {}` ("arrow notation") is a little shorter, if you want to use that instead